addons/web/static/tests/chrome/action_manager_tests.js

```diff
@@ -152,6 +154,11 @@ QUnit.module('ActionManager', {
             'pony,false,search': '<search></search>',
         };
     },
+    before: function () {
+        return Promise.all(_.map(WarningDialog.prototype.xmlDependencies, function (xmlPath) {
```

Comment:
you can load the file in test_utils.js, alongside other lazy loaded files (we don't want tests to behave differently, depending on the fact that other tests have been executed before or not).